CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.0
```

Comment:
Should we increase a major version for a breaking version?

Reply:
We could. I wanted to land first with the deprecated API and then a new major version increment without it.

We could go directly to the new version. It makes migration a little harder, but also makes the null safety upgrade a major version increment, which other package have done.

Reply:
How about push them both in the same time, and release a pre-release version? (3.0.0-nullsafety.0)